src/PeoplePage.tsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from 'react';
+
+import { Person } from './types/Person';
+import { getPeople } from './api';
+import { Loader } from './components/Loader';
+import { NoPeople } from './NoPeople';
+import { LoadingError } from './components/LoadError';
+import { TablePeople } from './components/TablePeople';
+
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [newError, setNewError] = useState(false);
+  const [load, setLoad] = useState(true);
+
+  useEffect(() => {
+    getPeople()
+      .then((peopleFromServer) => {
+        setLoad(true);
+        setPeople(peopleFromServer);
+      })
+      .catch((error) => {
+        setNewError(true);
+        throw error;
```

Comment:
Throwing an error after setting 'setNewError(true)' is not required as it stops the execution of the current function and will not allow for the '.finally' block to be executed and hence, 'setLoad(false)' would not be called.